packages/nvidia_nat_core/src/nat/llm/dynamo_llm.py

```
@@ -361,245 +377,211 @@ def get_dynamo_field_names() -> frozenset[str]:
+                        int(prediction.output_tokens.p90),
+                        iat,
+                        int(prediction.interarrival_ms.mean),
+                    )
+                else:
+                    logger.debug(
+                        "No prediction found for path=%s, call_index=%d; using static values",
+                        path,
+                        call_index,
+                    )
+
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)